tests/unit/test_opentelemetry_tracing.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import pytest
+
+from importlib import reload
```

Comment:
These tests are failing on Python 2. Need to use `six.moves.reload_module` https://six.readthedocs.io/#module-six.moves